mods/ts/rules/gdi-vehicles.yaml

```diff
@@ -278,6 +278,12 @@ SONIC:
 		Offset: -170,0,0
 	AutoTarget:
 	WithVoxelTurret:
+	Explodes:
+		RequiresCondition: !rank-elite
+	Explodes@ELITE:
+		RequiresCondition: rank-elite
+		Weapon: UnitExplode
+		EmptyWeapon: UnitExplode
```

Comment:
If Weapon and EmptyWeapon's default is UnitExplode, isn't it practice/better to not include it in .yaml?